install.sh

```diff
@@ -88,16 +91,25 @@ The installer will cache all downloaded assets into ~/.cache/coder
 EOF
 }
 
-echo_latest_version() {
-	if [ "${EDGE-}" ]; then
-		version="$(curl -fsSL https://api.github.com/repos/coder/coder/releases | awk 'match($0,/.*"html_url": "(.*\/releases\/tag\/.*)".*/)' | head -n 1 | awk -F '"' '{print $4}')"
-	else
-		# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c#gistcomment-2758860
-		version="$(curl -fsSLI -o /dev/null -w "%{url_effective}" https://github.com/coder/coder/releases/latest)"
-	fi
-	version="${version#https://github.com/coder/coder/releases/tag/}"
-	version="${version#v}"
-	echo "$version"
+echo_latest_stable_version() {
+	# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c#gistcomment-2758860
+	version="$(curl -fsSLI -o /dev/null -w "%{url_effective}" https://github.com/coder/coder/releases/latest)"
+	version="${version#https://github.com/coder/coder/releases/tag/v}"
+	echo "${version}"
```

Comment:
might be confusing with VERSION

Reply:
Alright, we can always adopt `local` keyword :)

Reply:
Caveat emptor: https://unix.stackexchange.com/questions/493729/list-of-shells-that-support-local-keyword-for-defining-local-variables#493743